lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Why would it fail? `{ all: true, include: ... }` is compatible with the `{ all: true }` interface. You could always add properties to the objects, TS and Sequelize wouldn't care